css/amp.css

```diff
@@ -167,6 +167,13 @@ html.i-amphtml-singledoc.i-amphtml-ios-embed-sd > body {
   display: inline-block;
 }
 
+
+.i-amphtml-blur {
+  filter: blur(20px) !important;
+  transform: scale(1.1) !important;
+  transition: opacity 0.3s cubic-bezier(0.0, 0.0, 0.2, 1) !important;
```

Comment:
We decided to use this cubic-bezier as it is what is suggested for easing elements in per the material design doc:
[https://material.io/design/motion/speed.html#easing](url)